### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by jillr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Gouttelette
+## _Cloud Content Code Generator_
+
+This repository contains the `gouttelette` package, a well tested python libraries that generates the cloud collections [``amazon.cloud``](https://github.com/ansible-collections/amazon.cloud) and [``vmware.vmware_rest``](https://github.com/ansible-collections/vmware.vmware_rest).
```
Comment (alinabuzachis) [inline README.md]: I guess this is a typo?
```suggestion
This repository contains the `gouttelette` package, a well tested python library that generates the cloud collections [``amazon.cloud``](https://github.com/ansible-collections/amazon.cloud) and [``vmware.vmware_rest``](https://github.com/ansible-collections/vmware.vmware_rest).
```
or
```suggestion
This repository contains the `gouttelette` package, a well tested python libraries that generate the cloud collections [``amazon.cloud``](https://github.com/ansible-collections/amazon.cloud) and [``vmware.vmware_rest``](https://github.com/ansible-collections/vmware.vmware_rest).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Apart from the above requirements, the tool needs
+
+- `Python 3.9`
+- `tox`
+
+### Usage
+
+The ``gouttelette`` generator is capable of performing the following functionalities
+
+- [`refresh_modules`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_modules.py) : Generates the modules specified in [resources.py](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/resources.py). It is important to make sure the rest schema and modules list files are present in the default location (details below), if not passed as input arguments.
```
Comment (alinabuzachis) [inline README.md]:
```suggestion
- [`refresh_modules`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_modules.py) : Generates the modules specified in [resources.py](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/resources.py). It is important to make sure the REST schema and modules list files are present in the default location (details below), if not passed as input arguments.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- Clone [``gouttelette repo ``](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette).
+```
+git clone https://github.com/ansible-collections/gouttelette
+```
+- From gouttelette execute the following command
+```
+python -m gouttelette.cmd.refresh_modules --collection "amazon_cloud"
+```
+The default values are taken for the other input arguments.
+Same steps are followed to refresh schema , refresh examples and refresh ignore files.
```
Comment (alinabuzachis) [inline README.md]:
```suggestion
Same steps are followed to refresh schema, refresh examples and refresh ignore files.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+```
+mkdir -p ~/.ansible/collections/ansible_collections/vmware/vmware_rest
+cd ~/.ansible/collections/ansible_collections/vmware/vmware_rest
+git clone https://github.com/ansible-collections/vmware.vmware_rest
+tox -e refresh_modules
+``` 
+
+We can specify the version
+```
+tox -e refresh_modules --vetsion "2.0.0"
```
Comment (alinabuzachis) [inline README.md]:
```suggestion
tox -e refresh_modules --version "2.0.0"
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+```
+## Contributing
```
Comment (alinabuzachis) [inline README.md:126]: I guess there's missing a paragraph explaining how to run units.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+git clone https://github.com/ansible-collections/gouttelette
+```
+- From gouttelette execute the following command
+```
+python -m gouttelette.cmd.refresh_modules --collection "amazon_cloud"
+```
+The default values are taken for the other input arguments.
+Same steps are followed to refresh schema , refresh examples and refresh ignore files.
+
+### _Referesh modules from amazon.cloud or vmware.vmware_rest_
```
Comment (alinabuzachis) [inline README.md:80]: Doesn't make more sense to have two separate paragraphs for each collection?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+- [`refresh_schema`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_schema.py): This command is for the amazon.cloud . It uses the AWS CloudFormation API to generate the rest schemas for the modules mentioned in the modules.yaml.
+```
+  python -m gouttelette.cmd.refresh_schema  --schema-dir <path>
+```
+| Input args | Description  |
+| ------ | ------ |
+| schema-dir | Location where the generated schemas will be stored (default: ./gouttelette/api_specifications/amazon_cloud |
+
+- [`refresh_examples`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_examples.py): Generates the examples by executing the integration tests and populates the EXAMPLE block of the module documentation.
```
Comment (alinabuzachis) [inline README.md]: I guess this is a bit unclear to me, what do you mean by "executing the integration tests "?

File: README.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+- [`refresh_modules`](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/refresh_modules.py) : Generates the modules specified in [resources.py](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/cmd/resources.py). It is important to make sure the rest schema and modules list files are present in the default location (details below), if not passed as input arguments.
+```
+  python -m gouttelette.cmd.refresh_modules --collection <collection_name> --target-dir <path> --modules <path> --next-version <version> --schema-dir <path>
+```
+
+| Input args | Description  |
+| ------ | ------ |
+| collection | The collection for which the modules are generated (values: amazon_cloud or vmware_rest, default: amazon_cloud) |
+| target-dir | Location of the target repository (default: ./cloud or ./vmware_rest) |
+| modules | Location of the [modules.yaml](https://github.com/ansible-collections/gouttelette/blob/main/gouttelette/config/amazon_cloud/modules.yaml) file (default: gouttelette/config/<collection>)|
```
Comment (alinabuzachis) [inline README.md]: I guess a small explanation on the purpose of the modules.yaml is missing.

**Comment by GomathiselviS** [comment]
recheck

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**